gcp/appengine/frontend_handlers.py

```diff
@@ -171,6 +171,17 @@
   return render_template('vulnerability.html', vulnerability=vuln)
 
 
+@blueprint.route('/<potential_vuln_id>')
+def vulnerability_redirector(potential_vuln_id):
+  """Convenience redirector for /VULN-ID to /vulnerability/VULN-ID."""
+  vuln = osv_get_by_id(potential_vuln_id)
+  if vuln:
+    return redirect(f'/vulnerability/{potential_vuln_id}')
```

Comment:
This isn't valid, because we already check it's a valid ID before we redirect.

That said, I'll add a regex match to be extra safe, and that can also save a Datastore read when the path is completely bogus.